### PR TITLE
Some cleanup and removal of unused fields

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

File: queries/imma-artists.sparql [hunk 1]
```diff
@@ -3,18 +3,15 @@ prefix xhtml: <http://www.w3.org/1999/xhtml#>
 PREFIX xyz:  <http://sparql.xyz/facade-x/data/>
 PREFIX fx:   <http://sparql.xyz/facade-x/ns/>
 
-select distinct ?artistUrl ?artistNickname
+SELECT DISTINCT ?artistUrl ?artistNickname
 WHERE {
-    SERVICE <x-sparql-anything:media-type=text/html,html.selector=#az-group,location=https://imma.ie/artists/> {
+    SERVICE <x-sparql-anything:media-type=text/html,html.selector=#	,location=https://imma.ie/artists/> {
```
Comment (enridaga) [inline queries/imma-artists.sparql]: The selector should ensure only links in scope are matched, why remove it?

**Review by enridaga — COMMENTED** [review]
Thanks for the pull request.

This is mostly formatting with the deletion of some lines that were not needed. I am not sure about the removal of the CSS selector on imma-artists.sparql, though

File: queries/imma-artists.sparql (hunk 1 above)
Comment (lolgab) [reply queries/imma-artists.sparql]: Sorry, it was a typo. I'm going to revert this change.

**Comment by lolgab** [comment]
Thank you for the review!
I deleted a selector by mistake. Now it should be fine.

**Comment by enridaga** [comment]
Thanks @lolgab  for contributing :)

**Pull request merged**